src/Microsoft.ML.StandardTrainers/FactorizationMachine/FactorizationMachineTrainer.cs

```diff
@@ -77,12 +77,6 @@ public sealed class Options : TrainerInputBaseWithWeight
             [TlcModule.SweepableFloatParam(1e-8f, 1f, isLogScale: true)]
             public float LambdaLatent = 0.0001f;
 
-            /// <summary>
-            /// Whether to normalize the input vectors so that the concatenation of all fields' feature vectors is unit-length.
-            /// </summary>
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Whether to normalize the input vectors so that the concatenation of all fields' feature vectors is unit-length", ShortName = "norm", SortOrder = 6)]
-            public bool Normalize = true;
```

Comment:
>Normalize [](start = 24, length = 9)

I would double check with @wschin

Reply:
yep, did

---
In reply to: [265801014](https://github.com/dotnet/machinelearning/pull/2964#discussion_r265801014) [](ancestors = 265801014)